src/Tribe/Settings_Tab.php

```diff
@@ -127,17 +151,26 @@ public function __construct( $id, $name, $args = [] ) {
 		// Parse args with defaults.
 		$this->args = wp_parse_args( $args, $this->defaults );
 
+		// Add ID and name to the args.
+		$this->args['id']   = $id;
+		$this->args['name'] = $name;
+
 		// Set each instance variable and filter.
-		$this->id   = apply_filters( 'tribe_settings_tab_id', $id );
-		$this->name = apply_filters( 'tribe_settings_tab_name', $name );
```

Comment:
While I can't find a use of either of these filters, we can't just remove them. Incorporate them into the new code, or deprecate them.

i.e. 
`$this->args['id']   = apply_filters( 'tribe_settings_tab_id', $id );`

Oh, and they need docblocks - no rest for the weary, eh? 😉

Reply:
Actually, not sure why we had these separate from the foreach below - all we need to do is set a default (`null`) id and name in `$defaults` - at least for name 🤔